tests/e2e-tests/utils/flows.js

```diff
@@ -7,7 +7,95 @@ const baseUrl = process.env.WP_BASE_URL;
 const WP_ADMIN_NEW_PRODUCT = baseUrl + '/wp-admin/post-new.php?post_type=product';
 const WP_ADMIN_WC_SETTINGS = baseUrl + '/wp-admin/admin.php?page=wc-settings&tab=';
 
+const SHOP_PRODUCT = baseUrl + '/product/';
+const SHOP_CART_PAGE = baseUrl + '/cart/';
```

Comment:
These are technically configurable in WC, right? Do we care? Or can we always just ensure that our test environment uses these same slugs?

Reply:
We will need to make sure that our test environment has the same slugs setup. Otherwise, we would need to check for all kinds of variations of URLs.